package.json

```diff
@@ -1,11 +1,11 @@
 {
-    "name":    "nekland-standard-edition",
+    "name": "nekland-standard-edition",
```

Comment:
The name of the package is more something like `nekland-neko-wiki` right ?

Reply:
This was the name already set, should I set it ?

Reply:
You should modify it. You know; little ninja fixes. :p